apps/router-e2e/__e2e__/dom-components/app/index.tsx

```diff
@@ -14,7 +14,7 @@ export default function Page() {
     <ScrollView style={{ flex: 1 }} contentContainerStyle={{ flexGrow: 1, padding: 56 }}>
       <TestCase name="Actions">
         <Actions
-          dom={{ autoSize: true }}
+          dom={{ stretchWebView: true }}
```

Comment:
Avoid mentioning "WebView" in the prop names, devs should avoid thinking about the underlying portal between React components.